tests/integration/tracing_test.go

```diff
@@ -132,6 +141,112 @@ func containsNodeListSpan(req *traceservice.ExportTraceServiceRequest) bool {
 	return false
 }
 
+// testStreamRPCTracing tests that Stream RPC calls are properly traced
+func testStreamRPCTracing(t *testing.T) {
```

Comment:
The test looks good. 

The only minor comment is that it's has very very similar implementation as `testUnaryRPCTracing`. Could you make them reuse the same test function with different hooks and filter (Unary operation vs Stream operation)?

Reply:
Please ignore the failures in `ci-etcd-robustness-release3x-amd64`, I will take care of them soon.